### PR TITLE
Minor healing chemical nerf.

### PR DESCRIPTION
**Pull request by Sparrowbird611**
After having a talk with Dream Crusha and TripleZeta, I thought I'd put an idea I had into action. Changes listed here

- Silver Sulfadiazine and Styptic Powder deal 0.6 stamina damage every time they metabolize.
- Synthlflesh deals one stamina damage for every one unit of real damage healed.

These should be minor nerfs to the chemicals, to make them a little less effective mid combat, but still useful.

### CODE REVIEW

**Comment by triplezeta** [comment]
:+1: Good changes

I've said this before but I'd prefer if silver sulf and styptic had proportional stamina damage dealt on splash/touch similar to how synthflesh is here. Otherwise, this is a PR I welcome with open arms. Death to patchchems.

**Comment by Sparrowbird611** [comment]
I may look at that idea, but for now I want to see how this works out first before I move on to that.

**Comment by Superyodeler** [comment]
> carbies
harmies
burnies

I hate this

**Comment by triplezeta** [comment]
dont worry he does too

**Comment by Sparrowbird611** [comment]
Can we all agree we hate this? I'm too lazy to change it though.

**Comment by tiramisuapimancer** [comment]
based

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -386,18 +388,21 @@
...
+			carbies.adjustStaminaLoss((harmies+burnies)*0.66)
 			if(show_message)
-				to_chat(M, "<span class='danger'>You feel your burns and bruises healing! It stings like hell!</span>")
-			SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "painful_medicine", /datum/mood_event/painful_medicine)
-			if(HAS_TRAIT_FROM(M, TRAIT_HUSK, "burn") && M.getFireLoss() < THRESHOLD_UNHUSK && (M.reagents.get_reagent_amount(/datum/reagent/medicine/synthflesh) + reac_volume >= 100))
-				M.cure_husk("burn")
-				M.visible_message("<span class='nicegreen'>A rubbery liquid coats [M]'s burns. [M] looks a lot healthier!")
+				to_chat(carbies, "<span class='danger'>You feel your burns and bruises healing! It stings like hell!</span>")
+			SEND_SIGNAL(carbies, COMSIG_ADD_MOOD_EVENT, "painful_medicine", /datum/mood_event/painful_medicine)
+			if(HAS_TRAIT_FROM(M, TRAIT_HUSK, "burn") && carbies.getFireLoss() < THRESHOLD_UNHUSK && (carbies.reagents.get_reagent_amount(/datum/reagent/medicine/C2/instabitaluri) + reac_volume >= 100))
```
Comment (MarkSuckerberg) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:401]:
```suggestion
			if(HAS_TRAIT_FROM(M, TRAIT_HUSK, "burn") && carbies.getFireLoss() < THRESHOLD_UNHUSK && (carbies.reagents.get_reagent_amount(/datum/reagent/medicine/synthflesh) + reac_volume >= 100))
```

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]

**Review by Superyodeler — CHANGES_REQUESTED** [review]
The stamina damage for silver sulf and styptic appears to get removed right after it is applied when there is no other stamina damage. My best guess is that it's not doing enough stamina damage to reset the regen countdown.

Also, please change carbies/harmies/burnies to something less annoying. Using VSCode, you can just select the variable and pick the change all occurrences option in the right click menu.

**Pull request closed without merging**